### ISSUE TITLE
clear event isn't available on iOS

### ISSUE THREAD

**Issue by andres-torres-marroquin** [mattermost/react-native-paste-input#12]
Hi! 👋 
      
Firstly, thanks for your work on this project! 🙂

Today I used [patch-package](https://github.com/ds300/patch-package) to patch `@mattermost/react-native-paste-input@0.5.0` for the project I'm working on.

clear event isn't available on iOS

Here is the diff that solved my problem:

```diff
diff --git a/node_modules/@mattermost/react-native-paste-input/src/ios.tsx b/node_modules/@mattermost/react-native-paste-input/src/ios.tsx
index 51534e8..cd18a4e 100644
--- a/node_modules/@mattermost/react-native-paste-input/src/ios.tsx
+++ b/node_modules/@mattermost/react-native-paste-input/src/ios.tsx
@@ -135,6 +135,18 @@ const PasteInput = forwardRef((props: PasteInputProps, ref) => {
         };
     }, []);
 
+    if (inputRef.current) {
+        inputRef.current.clear = function() {
+          viewCommands.setTextAndSelection(
+            inputRef.current,
+            mostRecentEventCount,
+            '',
+            0,
+            0,
+          );
+        }
+    }
+
     useImperativeHandle(ref, () => inputRef.current);
 
     const _onBlur = (event: NativeSyntheticEvent<TextInputFocusEventData>) => {
```

<em>This issue body was [partially generated by patch-package](https://github.com/ds300/patch-package/issues/296).</em>

**Comment by enahum** [comment]
@andres-torres-marroquin feel free to submit a Pr :)

**Comment by larkox** [comment]
Fixed in #15
